frontend/src/component/pages/admin/changelog/CreateChangeSection.scss

```diff
@@ -0,0 +1,48 @@
+@import 'src/component/styles/color';
+@import 'src/component/styles/layouts';
+
+.create-change-section {
+  .title {
+    margin: 0 auto;
+  }
+
+  .create-change-form {
+    display: flex;
+    flex-direction: column;
+
+    .change-summary {
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      margin-top: 20px;
+      justify-content: space-between;
+
+      .markdown-viewer {
+        padding: 0 10px;
+        background-color: $color-create-change-markdown-background;
+        overflow-y: auto;
+      }
```

Comment:
If the internal class name of text field or markdown viewer changed, this will break. Let's try not rely on the internal implementation of those elements.
I recommend wrapping text field and markdown viewer with div element and style that element instead. We also need to modify those element internally and make make them adapt to the width & height of the potential parent.

Reply:
How about this: Let's add a TODO for theming & CSS module. I'll start looking into them this week. Then we can plan for large scale refactor.

Reply:
We now support CSS module. This can be done by supporting a new style.